exercises/custom-set/CustomSet.pm

```diff
@@ -0,0 +1,77 @@
+package CustomSet;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(
+    new add remove is_empty
+    is_member size to_list
+    union intersect difference
+    is_disjoint is_equal is_subset
+);
+
+sub new {
+    my ($class, @members) = @_;
+    die "You need to implement this.";
```

Comment:
Maybe we could use [the ellipsis statement](https://perldoc.pl/perlsyn#The-Ellipsis-Statement) instead? (perhaps commenting one with this link)